src/H5Dchunk.c

```diff
@@ -2466,7 +2466,12 @@ H5D__chunk_cacheable(const H5D_io_info_t *io_info, H5D_dset_io_info_t *dset_info
     FUNC_ENTER_PACKAGE
 
     /* Sanity check */
+#ifdef H5_HAVE_PARALLEL
     HDassert(io_info);
+#else
+        (void)
+        io_info;
```

Comment:
lol - yay, formatter

Reply:
strange -  I wonder if there is a setting for that?